### DISCUSSION TITLE
7490 07.39 Inhaus UNTESTET --> testet :-)

### DISCUSSION THREAD

**Discussion by aus-sua** [Freetz-NG/freetz-ng#637] [Allgemein]
Moin ;-) ,

ich habe einmal eine freetz-NG-Firmware mit der Inhaus 07.39-99190 für die 7490 
erstellt und "gebrand".
Ein schneller Test hat nichts auffallendes ergeben:
DSL/LAN/WiFi arbeiten ohne Probleme.
DECT/Fon/ISDN habe ich jetzt nicht getestet, mir ist aber da nichts merkwürdiges
aufgefallen. 

Die Vorversion auf der Box sollte wohl mind. 07.29 sein, sonst gibt es per
Update über AVM- und auch per freetz-NG-WebIf eine Fehlermeldung
(freetz-NG: INSTALL_FILESYSTEM_CHECKSUM=4) und es wird dann kein
Update durchgeführt (also: ein direktes update per WebIf von 07.12 zu 07.39-xxx
funktionierte nicht).
Das ist aber wohl bei AVM bei den anderen Boxen bei deren Labor-Updates
auch so beschrieben.
Per ftp-Bootloader konnte ich es mangels direkten Zugriff auf die 7490 jetzt nicht testeten.
Ich habe es per AVM- und auch per freetz-NG-WebIF aufgespielt
(7.12-->7.29-->7.39inhaus), danach noch ein downgrade auf 07.12 (2. Firmware,),
per freetz-NG-WebIf, alles ohne Probleme/Fehler.

Brauchst Du davon da noch Log-Files und/oder besondere Tests?

**Comment by fda77** [comment]
Das UNTESTED steht einfach bei allen inhaus+labor, das ist nicht gerätespezifisch. Bei jedem make kann durch die juis-abfragen auch eine neuere/andere Version genommen werden.
Dazu ist avm's juis scheinbar marode, es werden oft falsche/veraltete Antworten gegeben - oder auch mal überhaupt keine. Siehe https://github.com/Freetz-NG/freetz-ng/commits/master/docs/juis/README.md
Ich hab zb die ganze Zeit für die 7490 nur "07.39-96428" bekommen, deine "07.39-99190" ist aber auch nicht (mehr?) die aktuellste, auf der Suche danach kam ich die "07.39-101273" von juis!
Bei der 7.5 gibts auch noch einige Problemchen, wie 
 - keine/veraltete sourcen
 - dnsmasq/bind/unbound startverhalten durch neuen multid-service
 - (webif) patches, bereits bekannt der minmax-spektrum patch

Davon dass man zuerst 7.29 installier haben muss hab ich bislang noch nichts gehört.

**Reply by fda77** [reply]
Hm, ich bekomme über die action die von github ausgeführt wird gar keine 7490 inhaus.
An einem normalen Computer gehts. Wahrscheinlich wieder so ein AVM-Ding: alles geht, nur nichts richtig

**Reply by aus-sua** [reply]
Ja, das juis ist wohl immer 'mal etwas "komisch", andererseits könnt ich mir auch vorstellen, daß 
AVM github z.B. IP-mäßig einfach "aussperrt" (quasi: wir möchten unsere inhaus-betas nicht
automatisch auf github wiederfinden...).
Meinen Fehler [FILESYSTEM_CHECKSUM] konnte ich auch nicht wieder sehen.
Ich habe das noch einmal probiert (Box wieder zurück auf 07.12 - und dann per AVM-webIf
direkt auf 07.39-99190) ging da dann ohne Probleme?!
Die neuere, 07.39-101273, muß ich erst noch erstellen, z.Z. kein PC, keine VM im "Zugriff"...

**Reply by fda77** [reply]
Ich glaube es gibt einen juis-server-cluster und avm hat vergessen alle zu updaten. Jenachdem an welchen man gerät kommt jedes mal ne andere antwort
